### PR TITLE
Updated deps

### PR DESCRIPTION
**Pull request by bonkey**
- Tied Nimble to v4 which is swift2.x compatible
- Switched to official SwiftyJSON on “swift2” branch

### CODE REVIEW

File: Cartfile.private [hunk 1]
```diff
@@ -1,14 +1,14 @@
 # Tests
 
 github "Quick/Quick"
-github "Quick/Nimble"
+github "Quick/Nimble" "v4.1.0"
```
Comment (hani-ibrahim) [inline Cartfile.private]: why?

**Comment by bonkey** [comment]
👍

File: Cartfile.private (hunk 1 above)
Comment (bonkey) [inline Cartfile.private]: 5.0 is swift3 only

**Comment by ahmed-shehata** [comment]
👍

**Pull request merged**